### PR TITLE
Atttribute form documentation link fix + QFieldCloud bit

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Review by suricactus — APPROVED** [review]

File: documentation/how-to/attributes-form.en.md [hunk 1]
```diff
@@ -183,7 +183,11 @@ can be based on the geometry or expression variables.
 
 For more information regarding storing information related to your
 position in object attributes, refer to the deticated
-`GNSS documentation<gnss_variables>`{.interpreted-text role="ref"}.
+[GNSS documentation](./gnss/).
+
+For QFieldCloud users, two variables can be used with default values:
```
Comment (m-kuhn) [inline documentation/how-to/attributes-form.en.md:188]: Just default values or also other expressions?

Comment (nirvn) [reply documentation/how-to/attributes-form.en.md:188]: Certainly with other expressions, but this is mentioned within the default values section in the overall context of attribute tables, so I don't think it's appropriate to mention symbology here :)

Comment (m-kuhn) [reply documentation/how-to/attributes-form.en.md:188]: It currently gives a slight impression of only being usable with default values. Maybe "expressions like default values" or something the like could be used?

Comment (nirvn) [reply documentation/how-to/attributes-form.en.md:188]: What about: "For QFieldCloud users, two variables can be used in expressions including attribute form's default values:..." ?

Comment (nirvn) [inline documentation/how-to/attributes-form.en.md:188]:
```suggestion
For QFieldCloud users, two variables can be used in expressions including attribute form's default values:
```

**Comment by nirvn** [comment]
@m-kuhn , OK to merge?

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**